RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -494,13 +494,39 @@ class ProjectLoader {
         List<String> errors = []
 
         // Earliest check for valid input and output directories. If they are not accessible or writeable.
-        // Start with the input directory
-        errors += checkDirForReadabilityAndExecutability(analysis.getInputBaseDirectory(), "input")
-        errors += checkDirForReadabilityAndExecutability(analysis.getOutputBaseDirectory(), "output")
 
-        // Out dir needs to be writable
-        if (!FileSystemAccessProvider.instance.isWritable(analysis.getOutputBaseDirectory()))
-            errors << (String) "The output was not writeable at path ${analysis.getOutputBaseDirectory()}."
+        // Check if the configuration values are both set, print out warning if one was set empty.
+        // If one is empty, fill with the other value, if both are empty, abort!
+        // Checks are done before the readability tests because we need to check for the raw configuration values. The next
+        // check will already use translated value
+
+        String valueInDir = analysis.configuration.configurationValues.get(ConfigurationConstants.CFG_INPUT_BASE_DIRECTORY, "").value
+        String valueOutDir = analysis.configuration.configurationValues.get(ConfigurationConstants.CFG_OUTPUT_BASE_DIRECTORY, "").value
+        
+        if (!valueInDir && !valueOutDir) {
+            errors << "Both the input and output base directories are not set. You must set at least inputBaseDirectory or outputBaseDirectory."
+
+        } else {
+
+            // Fill variable, if it is missing. Log a warning.
+            if(!valueInDir) {
+                logger.always("The input base directory is not set, take the path of the output base directory for it.")
```

Comment:
The message sounds like a request, but actually you are doing the "requested" change already in the code. Therefore 

"The input base directory is not set. Taking the path of the output base directory instead."